mobile/src/main/res/xml/preferences.xml

```diff
@@ -58,6 +58,40 @@
             android:summary="@string/settings_openhab_sslhost_summary"
             android:title="@string/settings_openhab_sslhost" />
     </PreferenceCategory>
+    <PreferenceCategory android:title="Presence">
+        <CheckBoxPreference
+            android:defaultValue="false"
+            android:key="default_openhab_presence_enable"
+            android:summary="Enable location-based presence reporting"
+            android:title="Enable" />
+        <EditTextPreference
+            android:defaultValue="isPresent"
+            android:dependency="default_openhab_presence_enable"
+            android:inputType="textNoSuggestions"
+            android:key="default_openhab_presence_item"
+            android:summary=""
+            android:title="Presence Item" />
+        <EditTextPreference
+            android:defaultValue=""
+            android:dependency="default_openhab_presence_enable"
+            android:inputType="numberDecimal"
+            android:key="default_openhab_presence_lat"
+            android:summary=""
+            android:title="Latitude" />
+        <EditTextPreference
+            android:defaultValue=""
```

Comment:
You could replace "" with "@string/empty_string"

Reply:
Not entire clear what you meant, but I just removed the default.